presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDictionarySelectiveReader.java

```diff
@@ -622,7 +622,16 @@ public void close()
     @Override
     public long getRetainedSizeInBytes()
     {
-        return INSTANCE_SIZE + sizeOf(currentDictionaryData) + sizeOf(values) + sizeOf(outputPositions);
+        return INSTANCE_SIZE +
+                sizeOf(values) +
+                sizeOf(outputPositions) +
+                sizeOf(currentDictionaryData) +
+                sizeOf(stripeDictionaryData) +
+                sizeOf(stripeDictionaryOffsetVector) +
+                sizeOf(stripeDictionaryData) +
+                sizeOf(stripeDictionaryLength) +
+                sizeOf(rowGroupDictionaryLength) +
+                sizeOf(evaluationStatus);
```

Comment:
I noticed we were missing accounting for these fields as well. However @bhhari thought that we may be double counting here. Am I missing something here?

Reply:
The `currentDictionaryData` may point to `stripeDictionaryData` therefore it should be removed. However other arrays should be included. cc @bhhari